clusters/threat-actor.json

```diff
@@ -1045,6 +1046,8 @@
           "https://unit42.paloaltonetworks.com/atoms/iron-taurus/"
         ],
         "synonyms": [
+          "GreedyTaotie",
+          "Emissary Panda",
```

Comment:
We don't need that, already had "EMISSARY PANDA" in the value field.

Reply:
oops my bad